packages/integration-tests/src/views/diskEncryption.js

```diff
@@ -0,0 +1,20 @@
+export const diskEncryptionPage = {
+  getTangServerUrl : (index) => {
+    return cy.get(`#form-input-diskEncryptionTangServers-${index}-url-field`);
+  },
+  getTangServerThumbprint : (index) => {
+    return cy.get(`#form-input-diskEncryptionTangServers-${index}-thumbprint-field`);
+  },
+  getMastersEncryptionSwitch: () => {
+    return cy.get('input[name="enableDiskEncryptionOnMasters"]');
+  },
+  getWorkersEncryptionSwitch: () => {
+    return cy.get('input[name="enableDiskEncryptionOnWorkers"]');
+  },
+  getEncryptionMode: () => {
+    return cy.get('input[name="diskEncryptionMode"]');
+  },
+  addAnotherTangServer: () => {
+    return cy.contains('Add another Tang server');
```

Comment:
This selector might be too broad

Reply:
I actually thought it's pretty specific, and I also told Celia that there's no other way to identify it specific.